North-Seattle-College/ad440-winter2022-thursday-repo
# Submit btn

This submit button gets the entry from the input box and shows it as an alert.
 To test:
- we need to have the input box set up.
- enter some text and hit the button.
- you should get an alert that shows your entry.
